code/src/Components/Details.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react/jsx-indent-props */
+/* eslint-disable indent */
+import React, { useState, useEffect } from 'react';
```

Comment:
Overall, the code looks clean and well-organized. The use of React hooks and functional components is great to see, and it's good to see the code is modularized with separate CSS files.